### PR TITLE
Add "Submitting your first conference talk" blog post

### PR DESCRIPTION
**Pull request by joshleecreates**
**Preview**: https://deploy-preview-2294--opentelemetry.netlify.app/blog/2023/submitting-your-first-conference-talk/

### CODE REVIEW

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Submitting Your First Conference Talk
+linkTitle: Submitting a CFP
+date: 2023-02-07
+author: '[Josh Lee](https://github.com/joshleecreates/) (IBM)'
+canonical_url: https://joshuamlee.com/submitting-your-first-conference-talk/
+---
+
+Submitting a conference talk in the tech industry can be a thrilling and nerve-wracking experience, especially if it’s your first time. The thought of presenting your ideas to a room full of experts and industry professionals can be intimidating, but also an opportunity to showcase your skills, gain recognition, and network with others in your field. 
+
```
Comment (avillela) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:10]:
```suggestion
On Wednesday, February 1st, 2023, the OpenTelemetry End User Working Group ran an Observability Abstract Workshop. The goal was to help folks who are new to speaking, and are interested in submitting an observability panel or talk. The workshop featured advice and expertise from OpenTelemetry community members [Reese Lee](https://github.com/reese-lee), [Daniel Kim](https://github.com/lazyplatypus), [Rynn Mancuso](https://github.com/musingvirtual), [Juraci Paixão Kröhling](https://github.com/jpkrohling), and [Adriana Villela](https://github.com/avillela).
```

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Whether you’re a seasoned speaker or just starting out, this guide will provide you with tips to help you make the most of your conference talk submission and increase your chances of being selected to speak.
```
Comment (avillela) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:11]:
```suggestion
What follows is a summary of the workshop's main takeaways. Whether you’re a seasoned speaker or just starting out, this guide will provide you with tips to help you make the most of your conference talk submission and increase your chances of being selected to speak.
```

**Review by chalin — COMMENTED** [review]
Hey @joshleecreates, can you sign the CLA. And then run Prettier over the file (`npx prettier --ignore-path '' -w <path-to-your-file>`)? Thanks!

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+* [GitHub - scraly/developers-conferences-agenda](https://github.com/scraly/developers-conferences-agenda)
+* [SeeCFP](https://seecfp.com/)
+* [Sessionize](https://sessionize.com/)
+
+## Writing your abstract
+Some conferences just require a few sentences or bullet points about your talk, while others may require a few pages of information including a detailed abstract and bio. In either case, remember to think about the abstract from the (potential) audience’s point of view. 
+
+It’s important to cover what technologies you will discuss, of course, but you also want to make sure you do a proper introduction to the use case for people who may not know the technology. 
+
+One of the most valuable talks I ever attended was about a topic I needed (debugging strategies) in a language I had never used at that point (.NET)
```
Comment (reese-lee) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:62]: Add a period to the end of this sentence.

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+canonical_url: https://joshuamlee.com/submitting-your-first-conference-talk/
+---
+
+Submitting a conference talk in the tech industry can be a thrilling and
+nerve-wracking experience, especially if it’s your first time. The thought of
+presenting your ideas to a room full of experts and industry professionals can
+be intimidating, but also an opportunity to showcase your skills, gain
+recognition, and network with others in your field. On Wednesday, February 1st,
+2023, the
+[OpenTelemetry End User Working Group](https://opentelemetry.io/community/end-user/)
```
Comment (chalin) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:15]:
```suggestion
[OpenTelemetry End User Working Group](/community/end-user/)
```

Comment (chalin) [reply content/en/blog/2023/submitting-your-first-conference-talk.md]: The change suggested above will fix the build.

**Comment by chalin** [comment]
Woohoo, we have a preview! ✨

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+to speaking, and are interested in submitting an observability panel or talk.
+The workshop featured advice and expertise from OpenTelemetry community members
+[Reese Lee](https://github.com/reese-lee),
+[Daniel Kim](https://github.com/lazyplatypus),
+[Rynn Mancuso](https://github.com/musingvirtual),
+[Juraci Paixão Kröhling](https://github.com/jpkrohling), and
+[Adriana Villela](https://github.com/avillela). What follows is a summary of the
+workshop's main takeaways. Whether you’re a seasoned speaker or just starting
+out, this guide will provide you with tips to help you make the most of your
+conference talk submission and increase your chances of being selected to speak.
```
Comment (reese-lee) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:26]: I know you just made this change -- I recommend breaking this up into at least 2 paragraphs, maybe even 3, for better readability, e.g.:

"Submitting..."

"On Wednesday..."

"What follows..."

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+## Tip #1: Just do it
+
+Don’t let perfect be the enemy of good. The best way to start giving talks is to
```
Comment (reese-lee) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:30]: perfection* ?

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+just start submitting and see what happens. You don’t need to craft the perfect
+proposal and you don’t need to be an industry-leading subject matter expert.
```
Comment (reese-lee) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:32]: Suggested edit: 

"You don’t need to craft the perfect proposal (although tailoring it for the event and target audience will certainly help!) and you don’t need to be an industry-leading subject matter expert."

Comment (joshleecreates) [reply content/en/blog/2023/submitting-your-first-conference-talk.md:32]: Thannks, great suggestions!

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+Don’t let perfectection be the enemy of good. The best way to start giving talks
+is to just start submitting and see what happens. You don’t need to craft the
+perfect proposal (although tailoring it for the event and target audience will
+certainly help!) and you don’t need to be an industry-leading subject matter
+expert.
+
+Don’t let any excuse stop you. So many conferences and events are desperate for
+anybody who simply has the **willingness** to speak and a little something to
+share.
```
Comment (SergeyKanzhelev) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:41]:
```suggestion
share. You may be surprised how happy people to hear real world experience from their peers.
```

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+But if you’re still unsure, you may want to consider…
+
+## Tip #2: Start at a Meetup or User Group
+
+If you’re not sure about your public speaking and want to start in a
+low-pressure environment, I would definitely recommend pitching any topics to a
+local meetup or user group.
```
Comment (SergeyKanzhelev) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:49]:
```suggestion
local meetup or user group. You can even treat your next team meeting as an opportunity to speak, paying attention how you present your thoughts.
```

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+one problem” — although both of those topics could be interesting!
+
+If you’re stumped for deeper ideas, consider:
+
+- Sharing a learning journey (see Tip #3)
+- Sharing a real world case study about how you used a specific technology to
+  solve a problem
+- Diving deep on a specific topic or technology
+- Doing a bake-off or round-up of comparable tools
+
```
Comment (SergeyKanzhelev) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:88]:
```suggestion

If your topic is touching some Open Source project - you can always ask for the feedback from the expert in the community. They may suggest an interesting angle for your talk.  You can even try to find a co-speaker. Don't be shy asking for help and form great connections in the community.
```

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+So really, what you want to tell your audience in your abstract is: what will
+they learn from attending your talk?
+
+## Making the most of your conference experience
+
+Making the most of a conference could be a topic for an entirely new blog post,
+but I want to share my top two most important tips:
+
+1. Wear comfortable shoes
+2. Leave room for serendipity (the hallway track)
```
Comment (SergeyKanzhelev) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:130]:
```suggestion
2. Leave room for serendipity (the hallway track)
3. Collect stats, make notes, and record feedback to motivate you to do it again
```

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+
+The first point should be obvious. Most of us are used to spending 10-12 hours a
+day on our butts. Shifting that weight to our feet for a week can be a lot.
+
+The second point is less obvious. At my first conference I felt that I was
+missing out any time I wasn’t in a session. Now I know better. While the talks
+are amazing for revving my mental engine, the “Hallway Track” is where the best
+conversations and connections happen.
+
```
Comment (SergeyKanzhelev) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:139]:
```suggestion

Third is important - any data that may help you remember your experience or be used to brag or convince your company to sponsor your next speaking engagement, are very valuable. You have done a lot of work, be proud of it.
```

**Review by SergeyKanzhelev — COMMENTED** [review]
I understand this blog post is from the workshop. I offer a few ideas to extend this. Feel free to reject

File: content/en/blog/2023/submitting-your-first-conference-talk.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+The only way to take advantage of this is if you leave some wiggle room in your
+schedule and aren’t afraid to take breaks when you need them.
+
+## Credits
+
+This post was inspired by the discussion in the OTel End User Working Group’s
+session on writing CFPs to help OTel community members submit talks for the
+upcoming conferences, including
+[Observability Day at KubeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/observability-day/).
```
Comment (svrnm) [inline content/en/blog/2023/submitting-your-first-conference-talk.md:148]: this has been stated in the introduction already, so you might consider removing it?

Comment (joshleecreates) [reply content/en/blog/2023/submitting-your-first-conference-talk.md]: I moved the specific call out to a new paragraph in the "Where to submit your talk session" — we could remove this next week after the deadline.

**Review by svrnm — APPROVED** [review]
lgtm

**Review by chalin — APPROVED** [review]
Other than the date change, which I think @svrnm will take care of, LGTM.

**Pull request merged**